src/main/java/anto/Ui.java

```diff
@@ -0,0 +1,192 @@
+package anto;
+
+import java.util.ArrayList;
+
+/**
+ * Ui handles all responses.
+ */
+public class Ui {
+
+    private TaskList taskList;
+
+    /**
+     * Creates a Ui class.
+     */
+    public Ui() {}
+
+    /**
+     * Sets Ui's taskList to the given one.
+     *
+     * @param taskList TaskList to set it to.
+     */
+    public void setTaskList(TaskList taskList) {
+        this.taskList = taskList;
+    }
+
+    /**
+     * Returns string that greets the user.
+     *
+     * @return String that greets the user.
+     */
+    public String greet() {
+        return "Hello I'm Anto\n"
+                + "What can I do for you?";
+    }
+
+    /**
```

Comment:
Description could probably be more accurate, like the string that leaves user a parting message, since it doesn't just say bye.